main.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const electron = require('electron');
+const ipc = electron.ipcMain;
+const app = electron.app;  // Module to control application life.
```

Comment:
https://github.com/atom/electron/issues/3708
Supposedly, a global installation of electron-prebuilt causes trouble when using `electron.app` convention. Removing it, things got fine. `require('app')` worked either way.
Lets go with the Quick Guide convention itself, the one you have used.